bundle/edu.gemini.catalog/src/main/java/jsky/catalog/TablePlotSymbol.java

```diff
@@ -569,6 +533,7 @@ public String getRatio() {
     }
 
     /** Return the symbol's ratio, evaluating the expression for the given row. */
+    @SuppressWarnings("EmptyCatchBlock")
```

Comment:
I think we should fix the empty catch blocks rather than disabling the warnings. Swallowing `Throwable` is not ok.